src/implementations/twig/components/breadcrumb-core/breadcrumb-core.story.js

```diff
@@ -0,0 +1,67 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import { withKnobs, text, optionsKnob } from '@storybook/addon-knobs';
+import withCode from '@ecl/storybook-addon-code';
+import {
+  getExtraKnobs,
+  tabLabels,
+  getLinkKnobs,
+  getComplianceKnob,
+} from '@ecl/story-utils';
+
+import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
+import dataSimple from '@ecl/specs-component-breadcrumb/demo/data--simple';
```

Comment:
Although it's correct we are not focused on the EU system at the moment, I personally don't see the reason of the lengthily explanation here.

```js
const system = process.env.STORYBOOK_SYSTEM
  ? process.env.STORYBOOK_SYSTEM
  : false;
```

Can be 

```js
const system = process.env.STORYBOOK_SYSTEM || 'ec;
```

And that's it.

Yes, we had already had discussions related to handling storybook instances for the 2 and regardless of whether 1 storybook will show 2 systems or 2 storybook applications will show 2 systems, having an enviornment variable for setting it and setting EC is default is good enough for now.

Reply:
Data structure in both EC/EU systems in ecl v2 are the same and contents is different.

Not sure why the story for this component is attempting to be system-aware, but I think it's not necessary because we haven't started working on this yet as far as I know.

I'd suggest we keep working with the EC spec as for the rest and remove this system-check in the story.